Library/Homebrew/utils/github.rb

```diff
@@ -682,17 +682,21 @@ def self.create_bump_pr(info, args:)
           EOS
         end
 
-        begin
-          url = create_pull_request(tap_remote_repo, commit_message,
-                                    "#{username}:#{branch}", remote_branch, pr_message)["html_url"]
-          if args.no_browse?
-            puts url
-          else
-            exec_browser url
-          end
+        url = begin
+          create_pull_request(tap_remote_repo, commit_message,
+                              "#{username}:#{branch}", remote_branch, pr_message)["html_url"]
         rescue *API::ERRORS => e
           odie "Unable to open pull request: #{e.message}!"
         end
+
+        # Attempt to enable automerge on the created PR if requested.
+        GitHub::API.run_gh_cli(args: ["pr", "merge", "--auto", "--merge", "--delete-branch", url]) if args.automerge?
```

Comment:
Is there a reason we do this rather than call the API directly like we do everywhere else?

Does this work with `HOMEBREW_GITHUB_API_TOKEN`?

Reply:
See also similar-ish code in Homebrew/core: https://github.com/Homebrew/homebrew-core/blob/f0c6e070add5312cb25639bcade628c21a6de2af/.github/workflows/publish-commit-bottles.yml#L211-L232

Reply:
@Bo98 pretty much because:
- we already have code to shell out to `gh`
- we already rely on `gh` for homebrew/core
- honestly for anything that's not performance-critical: I'd rather we pushed more of our code into `gh` calls than doing direct API access.

If you don't consider this a hard blocker: would rather land this as-is to validate the idea/workflow and anyone who wants to can move to GraphQL later if that's desired.

Reply:
I'm not entirely convinced it'll work in Homebrew/core CI as is, but have of course not tested it.